src/backend/singlenode/singlenode.cpp

```diff
@@ -57,7 +57,7 @@ std::size_t memory_size;
 namespace argo {
 	namespace backend {
 
-		void init(std::size_t size) {
+		void init(std::size_t argo_size, std::size_t cache_size){
```

Comment:
renamed parameters should also be renamed in the function body, currently this does not even compile

Reply:
addressed in 87ef1b1